### PR TITLE
Update Exchange-Teams-interact.md

### PR DESCRIPTION
**Pull request by MarcoCarampanta**
I feel the explanation was very confusing, and have also verified with multiple colleagues who had the same perception.
It mixed up the feature availability on Exchange Online Dedicated with Exchange mailboxes Onprem, as meetings+user photo are in fact available for on-prem as long as the Exch version is compliant.

Also, the text confirmed that connectors were not available for the last 2 scenarios, while the table showed "Yes".

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -24,28 +24,33 @@ How Exchange and Microsoft Teams interact
 
 For the full Teams experience, every user should be enabled for Exchange Online, SharePoint Online, and Office 365 Group creation.
 
-Users' Exchange mailboxes can be hosted online or on-premises. Users hosted on Exchange Online or Exchange Dedicated vNext can use all the features of Teams. They can create and join teams and channels, create and view meetings, call and chat, modify user profile pictures, add and configure connectors, tabs, and bots.
+Users' Exchange mailboxes can be hosted online or on-premises, however some features require for an hybrid deployment to be in place with your Office 365 tenant.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md]:
```suggestion
Users' Exchange mailboxes can be hosted online or on-premises. However, some features require a hybrid deployment to be in place with your Office 365 tenant.
```

File: Teams/Exchange-Teams-interact.md [hunk 2]
```diff
@@ -24,28 +24,33 @@ How Exchange and Microsoft Teams interact
...
 
-Users hosted on either Exchange Online Dedicated - Legacy, or Exchange on-premises, must be synchronized to Azure Active Directory for Office 365. They can create and join teams and channels, add and configure tabs and bots, and chat and call. However, they can’t modify user profile pictures, or add and configure connectors. They can receive messages from connectors configured by other users. For creating and viewing meetings, it's a mixed bag: Creating and viewing meetings is supported for Exchange 2016 cumulative update 3 (CU3) and above, but not for versions prior to Exchange 2016 CU3.
+Users hosted on Exchange Online or Exchange Dedicated vNext can use all the features of Teams. They can create and join teams and channels, create and view meetings, call and chat, modify user profile pictures, add and configure connectors, tabs, and bots.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md]:
```suggestion
Users hosted on Exchange Online or Exchange Dedicated vNext can use all the features of Teams. They can create and join teams and channels, create and view meetings, call and chat, modify user profile pictures, and add and configure connectors, tabs, and bots.
```

File: Teams/Exchange-Teams-interact.md [hunk 3]
```diff
@@ -24,28 +24,33 @@ How Exchange and Microsoft Teams interact
...
+
+Users hosted on Exchange Online Dedicated (Legacy) – must be synchronized to Azure Active Directory on Office 365. They can create and join teams and channels, add and configure tabs and bots, make use of the chat and calling features. However, they can’t modify profile pictures manage meetings, access outlook contacts, or manage connectors.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md]:
```suggestion
Users hosted on Exchange Online Dedicated (Legacy) must be synchronized to Azure Active Directory on Office 365. They can create and join teams and channels, add and configure tabs and bots, and make use of the chat and calling features. However, they can’t modify profile pictures, manage meetings, access outlook contacts, or manage connectors.
```

File: Teams/Exchange-Teams-interact.md [hunk 4]
```diff
@@ -24,28 +24,33 @@ How Exchange and Microsoft Teams interact
...
+
+Users with mailboxes hosted on-premises – They must be synchronized to Azure Active Directory. Can make use of all the features in the above scenario, but additionally they can also change the User Profile Picture, and manage meetings – With the requirement that the on-premise Exchange server is running a version above Exchange Server 2016 (Cumulative Update 3).
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md]:
```suggestion
Users with mailboxes hosted on-premises must be synchronized to Azure Active Directory. They can make use of all the features in the above scenario, but additionally they can also change the user profile picture and manage meetings, providing Exchange Server 2016 (Cumulative Update 3), or above, is running on-premises.
```

File: Teams/Exchange-Teams-interact.md [hunk 5]
```diff
@@ -24,28 +24,33 @@ How Exchange and Microsoft Teams interact
...
+
+For quick reference the table below should be very helpful for feature availability based on the Exchange Environment.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md]:
```suggestion
The following table provides a helpful quick reference to feature availability based on the Exchange environment.
```

File: Teams/Exchange-Teams-interact.md [hunk 6]
```diff
@@ -24,28 +24,33 @@ How Exchange and Microsoft Teams interact
...
 |---|---|---|---|---|---|---|---|---|---|---|---|---|---|
 |**Exchange Online**|Yes <sup>2</sup>|Yes <sup>2</sup>|Yes|Yes|Yes|Yes|Yes|Yes|Yes|Yes|Yes|Yes|Yes|
 |**Exchange Online Dedicated vNext**|Yes <sup>2</sup>|Yes <sup>2</sup>|Yes|Yes|Yes|Yes|Yes|Yes|Yes|Yes|Yes|Yes|Yes|
-|**Exchange Online Dedicated – Legacy** (Sync to Azure AD required)|Yes <sup>2</sup>|Yes <sup>2,3</sup>|Yes <sup>4|Yes|No|No|Yes|Yes|No|Yes <sup>5|Yes|Yes|Yes|
-|**Exchange on-premises** (Sync to Azure AD required)|Yes <sup>2</sup>| Yes <sup>2,3</sup> |Yes <sup>4|Yes|Yes (Exchange 2016 CU3+)|Yes (Exchange 2016 CU3+)|Yes|Yes|No|Yes <sup>5|Yes|Yes|Yes|
+|**Exchange Online Dedicated – Legacy** (Sync to Azure AD required)|Yes <sup>2</sup>|Yes <sup>2,3</sup>|Yes <sup>4|Yes|No|No|Yes|Yes|No|Yes <sup>5|Yes <sup>6|Yes|Yes|
+|**Exchange on-premises** (Sync to Azure AD required)|Yes <sup>2</sup>| Yes <sup>2,3</sup> |Yes <sup>4|Yes|Yes (Exchange 2016 CU3+)|Yes (Exchange 2016 CU3+)|Yes|Yes|No|Yes <sup>5|Yes <sup>6|Yes|Yes|
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md]:
```suggestion
|**Exchange On-premises** (Sync to Azure AD required)|Yes <sup>2</sup>| Yes <sup>2,3</sup> |Yes <sup>4|Yes|Yes (Exchange 2016 CU3+)|Yes (Exchange 2016 CU3+)|Yes|Yes|No|Yes <sup>5|Yes <sup>6|Yes|Yes|
```

File: Teams/Exchange-Teams-interact.md [hunk 7]
```diff
@@ -24,28 +24,33 @@ How Exchange and Microsoft Teams interact
...
 
 <sup>1</sup> Exchange 2016 CU3 and above supported  
 <sup>2</sup> eDiscovery and Legal Hold for compliance on channel messages is supported for all hosting options.  
 <sup>3</sup> Teams private chat messages are not yet supported for Legal Hold for this hosting option.
-
 <sup>4</sup> Retention will use a shadow mailbox for the online user to store messages. [Microsoft Teams Supports eDiscovery for Teams user in an Exchange Hybrid environment](https://techcommunity.microsoft.com/t5/Microsoft-Teams-Blog/Microsoft-Teams-supports-eDiscovery-for-Teams-users-in-a/ba-p/200009).
-
 <sup>5</sup> Teams users with on premise Exchange mailbox may use voicemail with Teams and receive voicemail messages in Outlook, but voicemail messages will not be available to view or play within the Teams client.
+<sup>6</sup> If one of the Owners for a given team can add connectors, everyone else will be able to, even if their mailboxes are homed on-premise.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md]:
```suggestion
<sup>6</sup> If one of the owners of a team can add connectors, everyone else in that team will be able to do so, even if their mailboxes are homed on-premises.
```

**Comment by scanum** [comment]
@LolaJacobsen Hi Lola, changes look correct and have been reviewed. Could you please confirm and merge? Thanks!

**Comment by scanum** [comment]
@LolaJacobsen, any updates on this? Thanks!

**Comment by LolaJacobsen** [comment]
@CarolynRowe - you're MS.Reviewer for this one. OK to merge?

**Comment by CarolynRowe** [comment]
I'm not sure why I'm listed as the MS.Reviewer on this one.  I'm not sure Exchange Dedicated vnext terminology is correct.  @dstrome, can you review this?  Otherwise, maybe we should ask a PM to review.

**Comment by scanum** [comment]
@dstrome Hi David! Could you please review? Thanks

**Comment by MarcoCarampanta** [comment]
any news on these edits?
@scanum , @dstrome , @CarolynRowe 

This is becoming a bit higher profile, and is more and more, a very used article.

**Comment by dstrome** [comment]
hi @scanum @CarolynRowe @MarcoCarampanta I'm taking a look at this PR now.

**Comment by dstrome** [comment]
The changes to the topic look great. I just need to confirm with the product team the changes in support statements regarding connectors and profile pictures. I'll update as soon as I hear back.

**Comment by dstrome** [comment]
Hi - just a quick update to say I'm still waiting for confirmation from the product team about the suggested changes. Will keep you updated as I know more.

**Comment by scanum** [comment]
@dstrome Hi David, any updates on this? Thanks!

**Comment by scanum** [comment]
@dstrome Hi David, any updates on this? Thanks!

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
hi all,

I sincerely apologize for the delay getting this resolved. The individual who can best answer this question is currently unavailable. I'm trying to find someone else who can confirm the changes. I'll update this conversation when I know more. Thanks very much for your patience.

**Pull request merged**